index.html

```diff
@@ -0,0 +1,152 @@
+            <div class="program-card">
+              <img src="./src/img/icons/program_icon_01.png" alt="lecture icon">
+              <h3 class="program-card-heading">Workshop</h3>
+              <p class="program-desc">Dance and share with the best milongueros dancer from the world Don't miss it!.</p>
+            </div>
+            <div class="program-card">
+              <img src="./src/img/icons/program_icon_02.png" alt="exhibition icon">
+              <h3 class="program-card-heading">Milonga Class</h3>
+              <p class="program-desc">Dance and share with the best milongueros dancer from the world Don't miss it!.</p>
+            </div>
+            <div class="program-card">
+              <img src="./src/img/icons/program_icon_03.png" alt="forum icon">
+              <h3 class="program-card-heading">Tango Class</h3>
+              <p class="program-desc">Dance and share with the best milongueros dancer from the world Don't miss it!.</p>
+            </div>
+            <div class="program-card">
+              <img src="./src/img/icons/program_icon_04.png" alt="workshop icon">
+              <h3 class="program-card-heading">Vals Class</h3>
+              <p class="program-desc">Dance and share with the best milongueros dancer from the world Don't miss it!.</p>
+            </div>
+            <div class="program-card">
+              <img src="./src/img/icons/program_icon_05.png" alt="ignite icon">
+              <h3 class="program-card-heading">Milonga</h3>
+              <p class="program-desc">Dance and share with the best milongueros dancer from the world Don't miss it!.</p>
+            </div>
+          </div>
+          <!-- programs-container -->
+          <a class="program-link desktop" href="#">See the whole program</a>
+          <button class="program-button">
+            <a class="program-link mobile" href="#">Join International Tango Festival</a>
+          </button>
+        </div>
+        <!-- cross bg -->
+      </section>
+      <!-- Program Section -->
+
+      <!-- dancers section -->
+      <section id="dancers">
+        <h2 class="dancers-heading">Guest dancers</h2>
+        <div class="line"></div>
+        <div class="dancers"> </div>
+      </section>
+      <!-- dancers section -->
+
+      <!-- sponsors section --> 
+      <section id="sponsors">
+        <h3 class="sponsors-heading">Sponsors</h3>
+        <div class="line"></div>
+        <div class="sponsors-logos">
+          <img class="google-logo" src="./src/img/sponsors/sponsor-5.png" alt="Google logo" width="200" height="200">
+          <img src="./src/img/sponsors/sponsor-1.png" alt="Airbnb logo" width="200" height="200">
+          <img src="./src/img/sponsors/sponsor-2.png" alt="GitHub logo" width="250" height="250">
+          <img src="./src/img/sponsors/sponsor-3.png" alt="Firefox logo" width="200" height="200">
+          <img src="./src/img/sponsors/sponsor-4.png" alt="Micorvese Logo" width="200" height="150">
+        </div>
```

Comment:
- To improve your site's accessibility, I'd advise you wrap these images, each in a `li` tag, and all `li` tags be wrapped in one `ul` tag. Please look into this 👍 👍 
- I noticed some of your images have different sizes from the others, please ensure that all images are of the same size so the section can look much more like the one in the design template 😄 

Your Project:
<img width="945" alt="tango home sponsor" src="https://user-images.githubusercontent.com/73607512/197243239-5fe7563d-2f30-413b-8a04-17d710a29447.PNG">

- F[ollowing HTML best practices](https://worldofdev.info/6-reasons-why-you-shouldnt-style-inline/#:~:text=Inline%20styles%20increase%20the%20size,include%20them%20in%20every%20paragraph.), I suggest you do not apply styling in your HTML code, please keep all styling in your CSS file. Please, try to fix this 👍 👍

Reply:
ok